aspnetcore/web-api/index.md

```diff
@@ -1,217 +1,224 @@
 ---
 title: Build web APIs with ASP.NET Core
 author: scottaddie
-description: Learn about the features available for building a web API in ASP.NET Core and when it's appropriate to use each feature.
+description: Learn the basics of writing web API code in ASP.NET Core.
+monikerRange: '>= aspnetcore-2.1'
 ms.author: scaddie
 ms.custom: mvc
-ms.date: 01/11/2019
+ms.date: 04/11/2019
 uid: web-api/index
 ---
+
 # Build web APIs with ASP.NET Core
 
-By [Scott Addie](https://github.com/scottaddie)
+By [Scott Addie](https://github.com/scottaddie) and [Tom Dykstra](https://github.com/tdykstra)
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/web-api/define-controller/samples) ([how to download](xref:index#how-to-download-a-sample))
+ASP.NET Core supports building RESTful services, also known as web APIs, using C#. To handle requests, a web API app uses controllers. *Controllers* in a web API app are classes that derive from `ControllerBase`. This article shows how to use controllers for handling API requests.
 
-This document explains how to build a web API in ASP.NET Core and when it's most appropriate to use each feature.
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/web-api/index/samples). ([How to download](xref:index#how-to-download-a-sample)).
 
-## Derive class from ControllerBase
+## ControllerBase class
 
-Inherit from the <xref:Microsoft.AspNetCore.Mvc.ControllerBase> class in a controller that's intended to serve as a web API. For example:
+A web API app has one or more controller classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. For example, the web API project template creates a Values controller:
```

Comment:
I had web API **app** here because we labeled the ToC that way and I thought it necessary to explain that that doesn't mean an API app can only serve APIs, but on second thought "app" isn't essential here.